### ISSUE TITLE
Allow Import of Existing Site

### ISSUE THREAD

**Issue by dbareis** [Automattic/studio#188]
### What

I have an existing site, what I'd really prefer is some sort of sync / upload changes process, but failing that I should be able to import a site or its backup.

### Why

I wish to try local development.  It is not so useful unless I can work on a full copy of my site. Ideally everything would be the same including plugins.

### How

A sync or import function, probably some way to specify what you want the import to take.  If sync some way to specify what gets synced (probably don't want everything).

**Comment by wojtekn** [comment]
Thanks @dbareis, it's on our radar! 🙂

**Comment by wojtekn** [comment]
@dbareis While we are exploring possibilities to add import and export features, I wanted to share that we had a positive experience with the [All in One WP Migration](https://wordpress.com/support/import/import-using-all-in-one-migration/) plugin for importing and exporting Studio sites.

**Comment by roblarsen** [comment]
@wojtekn I tried using All in One WP  Migration and am getting the below error

<img width="421" alt="image" src="https://github.com/Automattic/studio/assets/361421/7970310e-2260-45f7-a67e-c094a38a25e7">

The physical disk space available is 1.4TB so it's a limitation of the virtual disk. Is there any way to configure that? Let me know if you'd prefer I open a new issue.

**Comment by wojtekn** [comment]
@roblarsen would you mind checking what is the size of the site's directory?

**Comment by roblarsen** [comment]
> @roblarsen would you mind checking what is the size of the site's directory?

@wojtekn The wpress file used in the 992mb used in the above screen shot. The full site folder is similar- 994mb

**Comment by fluiddot** [comment]
I checked the values reported in PHP for the total and free size of the filesystem and don't match the ones from my machine. There shouldn't be any limitation on the mounted virtual filesystem regarding size, as they are created against the physical filesystem. However, it's likely that the total and free disk values obtained in PHP do not reflect the actual values from the system.

I opened https://github.com/Automattic/studio/issues/341 as a follow-up on this issue.
